traveler-swift-core/TravelerKit/Models/ParkingItemQuery.swift

```diff
@@ -49,11 +49,7 @@ public struct ParkingItemQuery {
     }
 
     func isValid() -> Bool {
-        if self.airportIATA == nil && self.boundingBox == nil {
-            return false
-        } else {
-            return true
-        }
+        return true // I think this is valid, will leave method here
```

Comment:
Why does it always return true?

Reply:
That method is a stub. For the moment the query is always valid, but there might be changes in the backed that will require us to validate the query.